### PR TITLE
Add link to Jetpack placements settings and more info to terms checkbox

### PR DESCRIPTION
**Pull request by rclations**
### Signup View
Screen Shot 2019-10-18 at 5 02 44 PM

### Prevented Signup View
Sites with an issue that prevents ads activation will show a notice like the following:
Screen Shot 2019-10-18 at 5 02 15 PM

### Signup Success View
Following signup, this is what the user will see:
Screen Shot 2019-10-18 at 5 02 58 PM

---
#### Changes proposed in this Pull Request

* Adds a notice with link where Jetpack sites can access placement controls.
* Adds a new like to the terms checkbox affirming that the user agrees to only post family-friendly content and not purchase non-human traffic.
* Allows Jetpack sites to activate WordAds
* Improves the instant activation toggle display

Companion patch in D34231-code

#### Testing instructions

* Take a jetpack site with a premium plan or above and navigate to Earn > Ads to ensure the form displays the same as the screenshot above.
* Make sure the placements link goes to the correct section.
* Try signing up with a wp.com site and make sure nothing else is broken :)

### CODE REVIEW

File: client/my-sites/earn/ads/form-settings.jsx [hunk 1]
```diff
@@ -213,6 +213,12 @@ class AdsFormSettings extends Component {
 		}, 0 );
 	}
 
+	jetpckPlacementControls() {
```
Comment (Aurorum) [inline client/my-sites/earn/ads/form-settings.jsx]: Typo? :)

```suggestion
	jetpackPlacementControls() {
```

(same typo with line 543 though)

Comment (rclations) [reply client/my-sites/earn/ads/form-settings.jsx]: good catch ;)

File: client/my-sites/earn/ads/form-settings.jsx [hunk 2]
```diff
@@ -213,6 +213,12 @@ class AdsFormSettings extends Component {
 		}, 0 );
 	}
 
+	jetpackPlacementControls() {
+		const linkHref = '/marketing/traffic/' + this.props.site.slug;
+
+		return <Card href={ linkHref }>Manage Ad Placements in Marketing Settings</Card>;
```
Comment (dbspringer) [inline client/my-sites/earn/ads/form-settings.jsx]: Few thoughts here:
1) Title case seems out of place with other <click to do thing> links
2) We could probably shorten this to "Manage ad placements"
3) Should we set this to the self-hosted Jetpack Settings -> Traffic tab for Jetpack sites? When we finally update Calypso/Jetpack to be equivalent we can set it to the marketing/earn tab, but right now that tab doesn't do much for JP sites.

**Comment by dbspringer** [comment]
Code is working well and as described, just a couple notes about copy/settings link.

File: client/my-sites/earn/ads/form-settings.jsx (hunk 2 above)
Comment (dbspringer) [reply client/my-sites/earn/ads/form-settings.jsx]: We decided to push #3 in to a separate/TBD update.

**Comment by cphilleo** [comment]
I'm not sure the `adsProgramName` variable is working correctly. It looks like it reads "Ads is the leading advertising...". For wp.com we should say "WordAds", and Jetpack should be "Jetpack Ads" throughout to be consistent.

Otherwise, looks good.

**Comment by rclations** [comment]
> I'm not sure the `adsProgramName` variable is working correctly. It looks like it reads "Ads is the leading advertising...". For wp.com we should say "WordAds", and Jetpack should be "Jetpack Ads" throughout to be consistent.
> 
> Otherwise, looks good.

"Jetpack Ads" is still WordAds, so I just simplified for now to be "WordAds" in the interest of getting this shipped - but feel free to change later.
